src/renderer/commands/createDrawObject.ts

```diff
@@ -108,6 +112,8 @@ export function createDrawObject(
                             lightIntensities[i]);
 
                         color += spec * lightColors[i];
+
+                        color += ambientLight;
```

Comment:
I think we should either divide this by the number of lights or put ambient light outside the loop so that it isnt affected by adding more point lights

Reply:
Makes sense, especially since it's being passed in to the renderer as a thing for the entire scene. Updated.